File: civilian38/2609.cpp
def gcd(num1, num2):
    a, b = max(num1, num2), min(num1, num2)
    if not a % b:
        return b
    else:
        return gcd(a % b, b)

a, b = tuple(map(int, input().split()))

print(gcd(a, b))
print(gcd(a, b) * (a // gcd(a, b)) * (b // gcd(a, b)))

File: h-j-lim/2609.py
def gcd(a, b):
    if a < b:
        return gcd(b, a)
    if b == 0:
        return a
    if a % b == 0:
        return b
    return gcd(b, a % b)


A, B = map(int,input().split())
print(gcd(A, B))
print((A * B) // gcd(A, B))
